### PR TITLE
HSC-24: Cleaned up concepts related to past history

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

**Comment by reagan-meant** [comment]
@rbuisson Need proof reading the concepts in french

File: configuration/concepts/Historique_sante_form.csv [hunk 1]
```diff
@@ -1,26 +1,12 @@
-uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1200
 2eac972d-9320-4570-b933-1435599d6885,,,Asthma health history,Asthme Historique de santé,Asthma,Asthme,,,Misc,Boolean,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:2]: I am guessing that those are existing diagnosis concepts right? Let's use them then.

File: configuration/concepts/Historique_sante_form.csv [hunk 2]
```diff
@@ -1,26 +1,12 @@
...
-6d1450bb-0109-456b-82f9-e495bcb5c959,,,Hopital de Trou du Nord,Hopital de Trou du Nord,Hopital de Trou du Nord,Hopital de Trou du Nord,,,Misc,N/A,,,,,,,,,,,
-57195bab-5935-4da4-b5de-e9ee1dba162e,,,Hopital de Pignon,Hopital de Pignon,Hopital de Pignon,Hopital de Pignon,,,Misc,N/A,,,,,,,,,,,
-26c0366b-83e4-4401-9699-e545219b8870,,,Other Hospital,Autre Hopital,Other,Autre,,,Misc,N/A,,,,,,,,,,,
-345e4897-ef95-4b67-a46a-27f995e9e5ed,,,Places,Lieux,Places,Lieux,,,Misc,Coded,,Hopital Justinien Cap Haitien;Hopital Sacré Coeur Milot;Hopital HBCH;Hopital St Jean Limbe;Hopital Bon Samaritain Limbe;Hopital de Trou du Nord;Hopital de Pignon;Other Hospital,,,,,,,,,
+276cccb6-0c0b-4f84-8947-298aceea5e62,,,Hospitals,Hôpitaux,Hospitals,Hôpitaux,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritan Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Other,,,,,,,,,
+c7a45d8f-3032-4499-92d2-5c4e13f1f824,,,Past Admission History,Antécédents d'admission,Past Admission History,Antécédents d'admission,,,ConvSet,N/A,,,Date;Number of Days;Reason;Hospitals,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:8]: "Historique d'hospitalisations"

File: configuration/concepts/Historique_sante_form.csv [hunk 3]
```diff
@@ -1,26 +1,12 @@
...
 f891f12b-9947-4cc4-aa97-b6ba47567a58,,,Allergies health history,Allergies Historique de santé,Allergies,Allergies,,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:9]: "Allergies"

File: configuration/concepts/Historique_sante_form.csv [hunk 4]
```diff
@@ -1,26 +1,12 @@
...
 61ac6631-a248-4dae-bd32-33520d26c04d,,,Tobacco use,Utilisation de tabac,Tobacco use,Utilisation de tabac,,,Misc,Boolean,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:10]: 'Consommation de tabac'

File: configuration/concepts/Historique_sante_form.csv [hunk 5]
```diff
@@ -1,26 +1,12 @@
...
 16a035ea-acf0-4e54-9c63-ffd126aaa3f7,,,Use of alcohol,Utilisation d'alcool,Use of alcohol,Utilisation d'alcool,,,Misc,Boolean,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:11]: 'Consommation d'alcool'

File: configuration/concepts/Historique_sante_form.csv [hunk 6]
```diff
@@ -1,26 +1,12 @@
...
 7d2e95f9-1368-4ef9-b65e-224aba05e5d3,,,Drug use,Utilisation de drogues,Drug use,Utilisation de drogues,,,Misc,Boolean,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/Historique_sante_form.csv:12]: 'Consommation de drogues'

**Review by rbuisson — CHANGES_REQUESTED** [review]
Few 'fr' changes.

File: configuration/concepts/Historique_sante_form.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/Historique_sante_form.csv]: This was an over-sight...Fixed

File: configuration/concepts/Historique_sante_form.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/Historique_sante_form.csv]: Fixed

**Pull request closed without merging**